tutorial_sarscov2.md

```diff
@@ -0,0 +1,231 @@
+
+First you'll need to prepare the data:
+
+ - For that test, you need to download from SRA the following runs:
+   [SRR10903401](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR10903401) and
+   [SRR10903402](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR10903402)
+   
+   (hint: click on the _data_ tab)
+ - Then arrange the downloaded file to have the following structure
+   (you need to rename the files so that the have `_R1` and `_R2` suffixes) :
+
+
+*[SRA]: [sequence read archive](https://trace.ncbi.nlm.nih.gov/Traces/sra/)
+
+```
+samples
+├── SRR10903401
+│   └── 20200102
+│       └── raw_data
+│           ├── wuhan2_R1.fastq
+│           └── wuhan2_R2.fastq
+└── SRR10903402
+    └── 20200102
+        └── raw_data
+            ├── wuhan1_R1.fastq
+            └── wuhan1_R2.fastq
+```
+
+## Miniconda
+
+V-pipe uses the [Bioconda](https://bioconda.github.io/)[^bioconda] bioinformatics software repository for all component of its pipeline.
+The pipeline itself is written using [snakemake](https://snakemake.readthedocs.io/)[^Snakemake].
+
+[^bioconda]: Grüning, Björn, Ryan Dale, Andreas Sjödin, Brad A. Chapman, Jillian Rowe, Christopher H. Tomkins-Tinch, Renan Valieris, the Bioconda Team, and Johannes Köster. 2018. “Bioconda: Sustainable and Comprehensive Software Distribution for the Life Sciences”. Nature Methods, 2018 doi:[10.1038/s41592-018-0046-7](https://doi.org/10.1038/s41592-018-0046-7).
+
+[^Snakemake]: Johannes Köster and Sven Rahmann. [Snakemake – a scalable bioinformatics workflow engine](https://academic.oup.com/bioinformatics/article/28/19/2520/290322). *Bioinformatics*, 28(19):2520–2522, 2012. doi:[10.1093/bioinformatics/bts480](https://doi.org/10.1093/bioinformatics/bts480)
+
+We're going to set up out environment:
+
+```bash
+# directory for this tutorial
+mkdir V-test
+cd V-test
+
+# Download Miniconda3
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+
+# -b for batch (no question asked)
+bash Miniconda3-latest-Linux-x86_64.sh -b -p ~/V-test/miniconda3
+```
+
+And now we're going to start miniconda and install snakemake from bioconda.
+
+```bash
+# mind the dot (=source)
+. ~/V-test/miniconda3/bin/activate
+
+conda create -n V-pipe -c bioconda snakemake conda
+conda activate V-pipe
+# We will let snakemake --use-conda handle
+# installation and download of V-pipe dependencies
+```
+> **Tips:**  To save space, you can install the *-minimal* version of snakemake, without all the GUI dependencies
+
+## Clone V-pipe
+
+
+```bash
+git clone -b sars-cov2 https://github.com/cbg-ethz/V-pipe.git
+cd V-pipe
+```
+> **Versions**: different version come with different defaults
+> - *sars-cov2* branch is adapted for SARS-CoV-2
+> - the default branch (*master*) is adapted for HIV
+
+## Test V-pipe
+
+Put the `samples` hierarchy that you created before in this `V-pipe` directory. Then verify that it has the [desired structure](#preparing-a-small-dataset) :
```

Comment:
Move/copy the samples directory you created in the step 'Preparing a small dataset' to this working directory. You can display the directory structure with 'tree samples' or 'find samples'.